xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemux.h

```diff
@@ -122,34 +123,29 @@ class CDemuxStream
 class CDemuxStreamVideo : public CDemuxStream
 {
 public:
-  CDemuxStreamVideo() : CDemuxStream()
-  {
-    iFpsScale = 0;
-    iFpsRate = 0;
-    iHeight = 0;
-    iWidth = 0;
-    fAspect = 0.0;
-    bVFR = false;
-    bPTSInvalid = false;
-    bForcedAspect = false;
-    type = STREAM_VIDEO;
-    iOrientation = 0;
-    iBitsPerPixel = 0;
-    iBitRate = 0;
-  }
+  CDemuxStreamVideo() { type = STREAM_VIDEO; };
 
   ~CDemuxStreamVideo() override = default;
-  int iFpsScale; // scale of 1000 and a rate of 29970 will result in 29.97 fps
-  int iFpsRate;
-  int iHeight; // height of the stream reported by the demuxer
-  int iWidth; // width of the stream reported by the demuxer
-  double fAspect; // display aspect of stream
-  bool bVFR;  // variable framerate
-  bool bPTSInvalid; // pts cannot be trusted (avi's).
-  bool bForcedAspect; // aspect is forced from container
-  int iOrientation; // orientation of the video in degrees counter clockwise
-  int iBitsPerPixel;
-  int iBitRate;
+  int iFpsScale = 0; // scale of 1000 and a rate of 29970 will result in 29.97 fps
+  int iFpsRate = 0;
+  int iHeight = 0; // height of the stream reported by the demuxer
+  int iWidth = 0; // width of the stream reported by the demuxer
+  double fAspect = 0; // display aspect of stream
+  bool bVFR = false;  // variable framerate
```

Comment:
remove the vertical alignment, https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#35-vertical-alignment